nipype/pipeline/plugins/multiproc.py

```diff
@@ -140,19 +151,28 @@ def __init__(self, plugin_args=None):
 
         # Instantiate different thread pools for non-daemon processes
         logger.debug('[MultiProc] Starting in "%sdaemon" mode (n_procs=%d, '
-                     'mem_gb=%0.2f)', 'non' * int(non_daemon), self.processors,
-                     self.memory_gb)
+                     'mem_gb=%0.2f, cwd=%s)', 'non' * int(non_daemon),
+                     self.processors, self.memory_gb, self._cwd)
 
         NipypePool = NonDaemonPool if non_daemon else Pool
         try:
-            self.pool = NipypePool(processes=self.processors,
-                                   maxtasksperchild=maxtasks)
+            self.pool = NipypePool(
+                processes=self.processors,
+                maxtasksperchild=maxtasks,
+                initializer=_init_worker,
```

Comment:
Is there a reason this shouldn't just be `os.chdir`?

Reply:
not really